webapp/app/components/roomList/roomList.component.js

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import { ifElse, equals, always } from 'ramda';
+import List, { ListItem, ListItemIcon, ListItemText } from 'material-ui/List';
+import ChatIcon from 'material-ui-icons/Chat';
+
+import { RoomLink } from './roomList.styles';
+
+
+export class RoomList extends PureComponent {
+  static propTypes = {
+    rooms: PropTypes.instanceOf(Map),
+    activeRoom: PropTypes.object.isRequired,
+  };
+
+  renderList = () => {
+    if (this.props.rooms.size) {
```

Comment:
jezeli przypiszesz defaultProps dla rooms to nie musisz sprawdzac size, wtedy wyrenderuje 0 elementow